pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -854,6 +854,11 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to add rule opening traffic from workers and masters")
 	}
+	log.Print("Allowing traffic from pods in same subnet")
```

Comment:
The comment is misleading - this is actually allowing all pod-pod traffic, which is fine. Note that all the subnets are connected through a router.